caiman/base/movies.py

```diff
@@ -1240,164 +1240,8 @@ def play(self,
-                        break
-            display(stopButton)
-            thread = threading.Thread(target=view, args=(stopButton,))
-            thread.start()
-
-        if fr is None:
-            fr = self.fr
-
-        looping = True
-        terminated = False
-        if save_movie:
-            fourcc = cv2.VideoWriter_fourcc(*opencv_codec)
-            frame_in = self[0]
-            if bord_px is not None and np.sum(bord_px) > 0:
-                frame_in = frame_in[bord_px:-bord_px, bord_px:-bord_px]
-            out = cv2.VideoWriter(movie_name, fourcc, 30.,
-                                  tuple([int(magnification * s) for s in frame_in.shape[1::-1]]))
-        while looping:
-
-            for iddxx, frame in enumerate(self):
-                if bord_px is not None and np.sum(bord_px) > 0:
-                    frame = frame[bord_px:-bord_px, bord_px:-bord_px]
-
-                if backend == 'opencv' or (backend == 'embed_opencv' and save_movie):
-                    frame = process_frame(iddxx, frame, bord_px, magnification, interpolation, minmov, maxmov, gain, offset, plot_text)
-                    if backend == 'opencv':
-                        cv2.imshow('frame', frame)
-                    if save_movie:
-                        if frame.ndim < 3:
-                            frame = np.repeat(frame[:, :, None], 3, axis=-1)
-                        frame = np.minimum((frame * 255.), 255).astype('u1')
-                        out.write(frame)
-                    if backend == 'opencv' and (cv2.waitKey(int(1. / fr * 1000)) & 0xFF == ord('q')):
-                        looping = False
-                        terminated = True
-                        break
-
-                elif backend == 'embed_opencv' and not save_movie:
-                    break
-
-                elif backend == 'pylab':
-                    im.set_data((offset + frame) * gain / maxmov)
-                    ax.set_title(str(iddxx))
-                    pl.axis('off')
-                    fig.canvas.draw()
-                    pl.pause(1. / fr * .5)
-                    ev = pl.waitforbuttonpress(1. / fr * .5)
-                    if ev is not None:
-                        pl.close()
-                        break
-
-                elif backend == 'notebook':
-                    logging.debug('Animated via MP4')
-                    break
-
-                else:
-                    raise Exception('Unknown backend!')
-
-            if terminated:
-                break
-
-            if save_movie:
-                out.release()
-                save_movie = False
-
-            if do_loop:
-                looping = True
-            else:
-                looping = False
-
-        if backend == 'opencv':
-            cv2.waitKey(100)
-            cv2.destroyAllWindows()
-            for i in range(10):
-                cv2.waitKey(100)
+        play(self, gain, fr, magnification, offset, interpolation, backend, do_loop, bord_px,
```

Comment:
Confusing to have an object method and a function both called play in the same file; maybe rename the function?

Reply:
Should I change it to `play_movie`? 
I noticed `estimates` also has a `play_movie` method, if for now the `embed_opencv` backend is the most convenient to play movies within a notebook, I'll add that backend to `estimates.play_movie` as well.

Reply:
play_movie() would be great.